evalml/tests/preprocessing_tests/test_k_means_smote_split.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.model_selection import StratifiedKFold, train_test_split
+
+from evalml.preprocessing.data_splitters import (
+    KMeansSMOTECVSplit,
+    KMeansSMOTETVSplit
+)
+
+im = pytest.importorskip('imblearn.over_sampling', reason='Skipping plotting test because imblearn not installed')
+
+
+def test_kmeans_smote_nsplits():
```

Comment:
You might need to drop some knowledge on me: what's the purpose of this?  I don't get why a splitter is hardcoded to return n_splits (and I think this is a personal problem of mine).

Reply:
I was just following convention of [TrainingValidationSplit](https://github.com/alteryx/evalml/blob/d92a3cb37f65a47119ac6a29820b404afc09825a/evalml/preprocessing/data_splitters/training_validation_split.py#L31). I think it's just additional information that could be helpful for users that we've provided previously.